CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.1.1
+
+## Enhancements
+
+* Update to Spark version 3.3.0.
```

Comment:
This doesn't matter, Spark is fully binary compatible for 3.x. So if you're releasing this just because of Spark 3.3, it's not necessary. However, I think we should note the package name change here.

Reply:
Yes, I added a not for the change of package name